Sources/App/routes.swift

```diff
@@ -7,6 +7,8 @@ func routes(_ app: Application) throws {
     }
 
     app.group("api") { api in
+        api.get("version") { req in API.Version(version: appVersion) }
```

Comment:
I typically add an endpoint that reports the deployed version back for easier confirmation - this will return a JSON response `{"version": <sha or tag>}`

Reply:
That's really interesting. Is there something you use that uses this?